spec/factories/tags.rb

```diff
@@ -3,5 +3,15 @@
     sequence(:oligo) { |n| ['A','C','G','T','A','C','G','T',n].shuffle[0,8].join }
     sequence(:group_id) { |n| n }
     tag_set_id { TagSet.find_or_create_by(name: 'Test Tag Set', uuid: '123456').id }
```

Comment:
Probably better to use a factory here. i.e. 
`create(:tag_set)`
I think you can just do
`tag_set { create(:tag_set) }`
The name of the tag set and the uuid are set as unique within the tag_set factory.

Reply:
I did comment on this once already, but probably in the wrong place on GitHub

I have updated to use create(:tag_set) but have had to define the relationship still using tag_set_id as otherwise some requests/v1 tests fail for a reason I cannot determine